halo/app/abci.go

```diff
@@ -69,16 +79,21 @@ func (l loggingABCIApp) ProcessProposal(ctx context.Context, proposal *abci.Requ
 	return resp, err
 }
 
-func (l loggingABCIApp) FinalizeBlock(ctx context.Context, block *abci.RequestFinalizeBlock) (*abci.ResponseFinalizeBlock, error) {
-	resp, err := l.Application.FinalizeBlock(ctx, block)
+func (l abciWrapper) FinalizeBlock(ctx context.Context, req *abci.RequestFinalizeBlock) (*abci.ResponseFinalizeBlock, error) {
```

Comment:
Does it make sense to change the variable name `l` -> `a`?
```suggestion
func (a abciWrapper) FinalizeBlock(ctx context.Context, req *abci.RequestFinalizeBlock) (*abci.ResponseFinalizeBlock, error) {
```